lkml/parser.py

```diff
@@ -330,20 +338,23 @@ def parse_block(self) -> Optional[dict]:
 
     @backtrack_if_none
     def parse_pair(self) -> Optional[dict]:
-        """Parse LookML key/value pair.
+        """Returns a dictionary that represents a LookML key/value pair.
+
+        Returns:
+            A dictionary with the parsed block or None if the grammar doesn't match
+
+        Grammar:
+            pair ← key value
 
         Examples:
-            Input (token stream):
+            Input (before tokenizing into a stream):
             ------
```

Comment:
nit: you removed the extra `-` under Output; there's also an extra under Input.

Reply:
Oh, I was trying to make these all a consistent 6 hyphens. Are `Input` and `Output` supposed to be different? Wasn't sure if there was a convention or if that was a typo.

Reply:
I see now, you were just underlining the word plus the space. I'll switch it back!